chart/Chart.yaml

```
@@ -19,7 +19,7 @@
 ---
 apiVersion: v2
 name: airflow
-version: 1.3.0-dev
+version: 1.3.0
```

```suggestion
version: 1.3.0-dev
```

We don't want to change this in main, right?